### ISSUE TITLE
When a gauge is in a repeated row, its query will not use the repeated variable

### ISSUE THREAD

**Issue by roidelapluie** [grafana/grafana#15739]
### What Grafana version are you using?

6.0.0

### What datasource are you using?

Prometheus

### What OS are you running grafana on?

linux

### What did you do?

- create a dashboard with a template variable, multi, include all values
- Add a repeated row
- add a gauge panel inside

### What was the expected result?

the query inside the gauge panel is using the "single" variable

```
http://127.0.0.1:3000/api/datasources/proxy/10/api/v1/query_range?query=count(up%7Bjob%3D~%22prometheus%22%7D)&start=1551483540&end=1551505200&step=60
http://127.0.0.1:3000/api/datasources/proxy/10/api/v1/query_range?query=count(up%7Bjob%3D~%22pushgateway%22%7D)&start=1551483540&end=1551505200&step=60
http://127.0.0.1:3000/api/datasources/proxy/10/api/v1/query_range?query=count(up%7Bjob%3D~%22alertmanager%22%7D)&start=1551483540&end=1551505200&step=60
http://127.0.0.1:3000/api/datasources/proxy/10/api/v1/query_range?query=count(up%7Bjob%3D~%22node%22%7D)&start=1551483540&end=1551505200&step=60
```

### What happened instead?

all values are used
(but the variable in the TITLE of the panel is correct!!)

```
http://127.0.0.1:3000/api/datasources/proxy/10/api/v1/query_range?query=count(up%7Bjob%3D~%22prometheus%7Cpushgateway%7Calertmanager%7Cnode%22%7D)&start=1551483540&end=1551505200&step=60
http://127.0.0.1:3000/api/datasources/proxy/10/api/v1/query_range?query=count(up%7Bjob%3D~%22prometheus%7Cpushgateway%7Calertmanager%7Cnode%22%7D)&start=1551483540&end=1551505200&step=60
http://127.0.0.1:3000/api/datasources/proxy/10/api/v1/query_range?query=count(up%7Bjob%3D~%22prometheus%7Cpushgateway%7Calertmanager%7Cnode%22%7D)&start=1551483540&end=1551505200&step=60
http://127.0.0.1:3000/api/datasources/proxy/10/api/v1/query_range?query=count(up%7Bjob%3D~%22prometheus%7Cpushgateway%7Calertmanager%7Cnode%22%7D)&start=1551483540&end=1551505200&step=60
```

### If related to metric query / data viz:

You can load this dashboard:
https://gist.github.com/roidelapluie/1a4f837d48d96e85964ff37e25e246ef

Screenshot (gauge should show 1):
![gauge](https://user-images.githubusercontent.com/291750/53677780-c0a07880-3cb5-11e9-94f3-ca633ec6a161.png)

**Comment by roidelapluie** [comment]
Note: it might affect all new react panels

**Comment by torkelo** [comment]
thanks for reporting this!

**Comment by roidelapluie** [comment]
It is panel/gauge or grafana/ui bug ?

**Comment by roidelapluie** [comment]
I tried to look in the code but I would expect the panel to received the correctly scoped variables.

**Comment by torkelo** [comment]
Nothing to do with Gauge specifically but probably a problem in DataPanel component that handed query execution

**Comment by roidelapluie** [comment]
e.g. https://github.com/grafana/grafana/blob/0114ffc9ac82af217263fc7abf49bc525ce59cb7/public/app/features/dashboard/dashgrid/DataPanel.tsx#L116 :)

**Comment by inakimalerba** [comment]
Hi there ! Was this included into 6.0.1 ?

**Comment by roidelapluie** [comment]
No, 6.1.0
